warrior/Framework/ClassUtils/json_utils_class.py

```diff
@@ -252,16 +253,18 @@ def compare_json_using_jsonpath(self, response, list_of_jsonpath, list_of_expect
         status = True
         json_response = json.loads(response)
         for index, jsonpath in enumerate(list_of_jsonpath):
-            json_path = jsonpath.strip("json=")
+            json_path = jsonpath.strip("jsonpath=")
             value = self.get_value_for_nested_key(json_response, json_path)
-            if value != list_of_expected_api_responses[index]:
+            # Check if the expected value/pattern is in actual response
+            match = re.search(list_of_expected_api_responses[index], value)
+            if not match:
```

Comment:
Hi @venkat1077,

Can we do both, the equality check and regex searching - so as to avoid a backward incompatibility?

Thanks,
Sanika

Reply:
Made the required changes.